src/System.Memory/src/System/Buffers/Text/Utf8Parser/Utf8Parser.Float.cs

```diff
@@ -68,7 +68,7 @@ public static bool TryParse(ReadOnlySpan<byte> source, out double value, out int
             if (TryParseNormalAsFloatingPoint(source, out value, out bytesConsumed, standardFormat))
                 return true;
 
-            return TryParseAsSpecialFloatingPoint<double>(source, double.PositiveInfinity, double.NegativeInfinity, double.NaN, out value, out bytesConsumed);
+            return TryParseAsSpecialFloatingPoint(source, double.PositiveInfinity, double.NegativeInfinity, double.NaN, out value, out bytesConsumed);
```

Comment:
The float version of this method specifies the generic type explicitly - they should stay consistent. (And I'd prefer you keep it explicit. We don't need to jump to accomodate every time the VS IDE whines that the compiler can figure something out. We don't write source code just for the compiler's sake.)